build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   ws,
   specs2 % Test,
   "com.gu" %% "pan-domain-auth-play_2-4-0" % "0.2.7",
-  "com.amazonaws" % "aws-java-sdk" % "1.7.5"
+  "com.amazonaws" % "aws-java-sdk" % "1.10.50"
```

Comment:
> specify only what you actually need from aws instead of loading the entire sdk

Agreed. `AmazonCloudWatchAsyncClient` and `AmazonEC2Client` is [needed](https://github.com/guardian/login.gutools/blob/master/app/config/aws/package.scala#L4-L7) - we could include `aws-java-sdk-core`, `aws-java-sdk-ec2` and `aws-java-sdk-cloudwatch`, however including the sdk as a whole feels like less maintenance. What do you think?

> Why not upgrading panda as well as your making some changes?

:+1: Updating to lastest panda means the diff of this PR is smaller! :smile: